index.html

```diff
@@ -1150,15 +1150,15 @@ <h2 id="relationship-to-permissions-policy">
         a HTTP header or a the [^iframe/allow^] attribute). The APIs and features in scope for the
         Permissions Policy specification go beyond those identified in this specification's
         {{PermissionName}} enum (e.g., "sync-xhr" and "gamepad"). In that sense, the Permissions
-        Policy subsumes this specification in that Permissions Policy governs wether a feature is
+        Policy subsumes this specification in that Permissions Policy governs weather a feature is
```

Comment:
Almost! :)

The correct spelling is "whether" in this instance.

Reply:
Oh, :D I made a typo while fixing a typo. Sorry about that. I will laugh awhile at myself, so I am a bit out now.